pkgs/tools/admin/ansible/default.nix

```
@@ -1,72 +1,27 @@
-{ stdenv, fetchurl, python2
-, windowsSupport ? false
-}:
+{ python3Packages, fetchurl }:
 
-let
-  generic = { version, sha256, py ? python2 }: py.pkgs.buildPythonPackage rec {
-    pname = "ansible";
-    inherit version;
+rec {
+  ansible = python3Packages.toPythonApplication python3Packages.ansible;
+
+  ansible_2_7 = ansible;
 
-    outputs = [ "out" "man" ];
+  ansible_2_6 = python3Packages.ansible.overridePythonAttrs(old: rec {
```

```suggestion
  ansible_2_6 = with python3Packages toPythonApplication (ansible.overridePythonAttrs (old: rec {
```
Not sure if the added parantheses is necessary.

---

Fixed